NYPL/sfr-bookfinder-front-end
# SFR-1057 Add QA Deployment step on feature branches

[Jira Ticket](http://jira.nypl.org/browse/SFR-1057)

### This PR does the following:
- Creates a temporary deployment of the current feature branch in the QA ECS cluster and provides a link as a comment on the PR

### Testing requirements & instructions: 
-  Verify that a new environment is created and available at the IP address provided in the PR comments
- Verify that updated code is deployed and front end can connect to API running in service
